Src/PerformanceCollector/Filtering/CollectionConfigurationAccumulator.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+
+    /// <summary>
+    /// Represents the part of the QuickPulse accumulator which holds operationalized metric data.
```

Comment:
Revised the comment making it clear that it is the actual collection-time data that is being accumulated.